src/pyramid/view.py

```diff
@@ -201,6 +201,16 @@ def my_view(context, request):
         ``view_config`` will work ONLY on module top level members
         because of the limitation of ``venusian.Scanner.scan``.
 
+        Also it will ONLY work on views present as plaintext python
+        files (*.py), ignoring all non-*.py ones which also includes
+        *.pyc and *.pyo (python precompiled byte code / optimized) files.
+
+        Another venusian specific precondition to take into account is that
+        python packages to be found by venusian need to have a __init__.py
+        file in their root directory.
+
+        This is a limitation in venusian documented in:
+        https://docs.pylonsproject.org/projects/venusian/en/latest/#using-venusian
```

Comment:
This paragraph should get moved up underneath the comment about `*.py{c,o}` because the Venusian docs don't currently warn about `__init__.py` being required.